### DISCUSSION TITLE
Facing issues wihile trying to connect to controller & simulatoor device via wensocket

### DISCUSSION THREAD

**Discussion by praveen2211** [OktopUSP/oktopus#132] [General]
@leandrofars I tried to connect to simulator device & controller via websocket by changing few params in websockets_factory_reset_example file but we are facing below errors not sure if any other params needs to be changed.

Error  - 
ServiceInactiveWebsockServer: Failed to restart WebSocket server. Retrying in 5 seconds
Attempting to connect to host=localhost (port=8080, unencrypted, path=/) from interface=eth0
HandleWscEvent_Error: Error (HS: ws upgrade response not 101) occurred on websocket connection to endpoint_id=os::4851CF-000000000002 (host=localhost)
HandleWscEvent_Destroyed: Closed connection to os::4851CF-000000000002
ScheduleWsclientRetry: Retrying connection in 122 seconds (retry_count=5)
StartWebsockServer: lws_create_context() failed
ServiceInactiveWebsockServer: Failed to restart WebSocket server. Retrying in 5 seconds
StartWebsockServer: lws_create_context() failed
ServiceInactiveWebsockServer: Failed to restart WebSocket server. Retrying in 5 seconds
StartWebsockServer: lws_create_context() failed
ServiceInactiveWebsockServer: Failed to restart WebSocket server. Retrying in 5 seconds
StartWebsockServer: lws_create_context() failed

**Comment by leandrofars** [comment]
@praveen2211 Oktopus don't support device websockets connection yet, link of the issue: https://github.com/OktopUSP/oktopus/issues/100

**Comment by leandrofars** [comment]
@praveen2211 It's scheduled to be done in December first.
